drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -509,36 +488,38 @@ function DragAndDropTemplates(configuration) {
             item_bank_properties.attributes = {
                 'tabindex': 0,
                 'dropzone': 'move',
-                'aria-dropeffect': 'move',
-                'role': 'button'
+                'aria-dropeffect': 'move'
             };
         }
+        var main_element_properties = {
+            attributes: {
+                'role': 'group',
+                'aria-label': gettext('Drag and Drop Problem')
```

Comment:
Did @cptvitamin request this change? Instead of setting a hard-coded label, I would use aria-labelledby and reference the problem-title.